docs/Guide.Mocha.md

```diff
@@ -0,0 +1,35 @@
+# Mocha setup guide
+
+This guide describes how to install [Mocha](mochajs.org) as a test runner to be used by Detox for running the E2E tests.
```

Comment:
I would mention that Mocha is a less preferred test runner to use, for two reasons. The first being its lack of multi worker support, and the second is the much more advanced integration with Jest Circus, and also some behaviours or bugs are only solvable with Circus.

Reply:
So if you want a 2018 retro feeling of Detox, use Mocha :)

Reply:
I couldn't agree more, but will take care of it on a different scope 🙏